code/parachain/runtime/common/src/lib.rs

```diff
@@ -217,21 +219,24 @@ pub fn multi_existential_deposits<AssetsRegistry>(_currency_id: &CurrencyId) ->
 	Balance::zero()
 }
 
+/// Given a `currency_id`, returns the existential deposit of a MultiAsset.
+/// Returns `1_000_000` as the existential deposit if unable to get an existential deposit for the
```

Comment:
Does it not make sense to just error out for non-existing ED?

Reply:
no. it should be fixed to tranfer ksm and usdt.